docs/introduction/using-form-types.md

```diff
@@ -108,6 +109,16 @@ DatePickerType has 2 constants that can be used:
 - FORMAT_PHP = 'dd.MM.yyyy'
 - FORMAT_JS = 'dd.mm.yy'
 
+### [DateTimeType](../../packages/framework/src/Form/DateTimeType.php)
+Displays a text field that allows to set date and time.  
+Value is internally converted from [display time zone](./working-with-date-time-values.md) to UTC.
+
+#### format
+Defaults to `DateTimeType::FORMAT_PHP`.  
+Defines in what format should be the date shown.
+DatePickerType has constant that can be used:
+- FORMAT_PHP = 'dd.MM.yyyy HH:mm:ss'
```

Comment:
Oh here it is... should not this be stored in parameters so implementators can change it easily?

Reply:
I consider it outside the scope because then I would need to change DatePickerType too (would I?).

Reply:
Hmmm maybe yes. Can you at least create issue for that?

Reply:
It actually exists - #152 (Dates on the frontend are not localized)
And we already have the User Story for that too (internal number 6364)